feat(curriculum): add string method workshop (#66936)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/workshop-employee-profile-validator/6a73427af41619a909fb21b4.md

---
id: 6a73427af41619a909fb21b4
title: Step 21
challengeType: 20
dashedName: step-21
---

# --description--

> Now print both variables using f-strings with the prefixes `'Dots in email: '` and `'Dashes in code: '`.

Reviewer: A camper opening this step gets a fresh seed, so "both variables" has no antecedent in the prose, while both hints grade `dot_count` and `dash_count` by name.

```suggestion
Now print `dot_count` and `dash_count` using f-strings with the prefixes `'Dots in email: '` and `'Dashes in code: '`.
```

With that, your employee profile validator workshop is complete!

# --hints--

> You should print `dot_count` with the prefix `'Dots in email: '` using an f-string.

Reviewer: Blank line directly after `# --hints--`.

```suggestion
# --hints--

You should print `dot_count` with the prefix `'Dots in email: '` using an f-string.
```

---
id: 6a73427af41619a909fb21b4
title: Step 19
challengeType: 20
dashedName: step-19
---

# --description--

Now print both variables using f-strings with the labels `"Dots in email: "` and `"Dashes in code: "`.

With that, your employee profile validator workshop is complete!

# --hints--

> You should print `dot_count` with the label `'Dots in email: '` using `f string`.

Reviewer 2: ```suggestion
You should print `dot_count` with the label `'Dots in email: '` using an f-string.
```

```js
({ test: () => assert(runPython(`_Node(_code).has_call("print(f'Dots in email: {dot_count}')")`)) })
```

> You should print `dash_count` with the label `'Dashes in code: '` using `f string`.

Reviewer 2: ```suggestion
You should print `dash_count` with the label `'Dashes in code: '` using an f-string.
```
